_layouts/documentation.html.slim

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Let's call this file _layouts/docs.html.slim. That way, it matches the folder name.